src/main/resources/xml/schema/2020-1/nordic2020-1.opf.sch

```diff
@@ -148,8 +148,11 @@
         <p></p>
         <rule context="opf:itemref[../../opf:manifest/opf:item[@media-type='application/xhtml+xml' and ends-with(@href,'-cover.xhtml')]/@id = @idref]">
             <let name="context" value="concat('(&lt;', name(), string-join(for $a in (@*) return concat(' ', $a/name(), '=&quot;', $a, '&quot;'), ''), '&gt;)')"/>
-            <assert test="@linear = 'no'">[opf10] Cover must be marked as secondary in the spine (i.e. set linear="no" on the itemref with idref="<value-of select="@idref"/>", which refers to the
-                cover)</assert>
+            <assert test="@linear = 'no'">[opf10] Cover must be marked as secondary in the spine (i.e. set linear="no" on the itemref with idref="<value-of select="@idref"/>", which refers to the cover)</assert>
+            <assert test="count(preceding-sibling::opf:itemref[not(@linear='no')]) = 0">[opf10] All preceeding documents to the cover must be marked as secondary in the spine (i.e. set linear="no")</assert>
+        </rule>
+        <rule context="opf:itemref[../../opf:manifest/opf:item[@media-type='application/xhtml+xml' and ends-with(@href,'titlepage.xhtml')][1]/@id = @idref]">
```

Comment:
Checking for `titlepage.xhtml` will probably work in all cases, so let's accept this here.

Note however that there could potentially be a hyphen + number between the role/type and the file extension. At least in the old guidlines. I've asked the guidelines group about it here: https://github.com/nlbdev/epub3-guidelines-update/issues/86#issuecomment-788952978

There's not going to be more than one titlepage though, so this is fine here. But for chapters etc. I think we should handle `(-\d+)?.xhtml` as well.

Reply:
Also, it just occured to me, that this will also match halftitlepage. I don't know if that was intended or not, but it's fine.

Reply:
Yes, that is correct and it was my intention to allow the halftitlepage as well. I was thinking about a good way to indicate this but the best / simplest way was just to check the last part without the dash.